src/app/league/page.tsx

```diff
@@ -0,0 +1,214 @@
+// app/league/page.tsx
+"use client";
+
+import * as React from "react";
+
+import { Calendar, BookOpen, Crown } from "lucide-react";
+
+import LeagueSectionCard from "@/components/league-section-card/index";
+
+import { useLeague } from "@/app/league/hooks";
```

Comment:
```suggestion
import { useLeague } from "./hooks";
```
同じ階層の関数とかは相対パスで呼んだ方がいいかも。